src/QandaAtom.tsx

```diff
@@ -1,7 +1,445 @@
 import React from 'react';
-
 import { QandaAtomType } from './types';
+import { css } from 'emotion';
+import { neutral, error } from '@guardian/src-foundations/palette';
+import { headline, textSans, body } from '@guardian/src-foundations/typography';
+
+const { useState } = React;
+
+// CSS
+const detailStyling = css`
```

Comment:
Can I suggest that this is targeted to the body rather than the detail element - so that you're more closely targeting the dangerouslySetHtml part (where you have to style elements) vs the outer elements  you have control over that you can style with className more easily